packages/terra-tag/tests/nightwatch/TagTests.jsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { Link } from 'react-router';
+
+const TagTests = () => (
+  <div>
+    <ul>
+      <li><Link to="/tests/tag-tests/default-tag">Tag - Default</Link></li>
+      <li><Link to="/tests/tag-tests/default-on-click-tag"> Tag - Defult OnClick</Link></li>
```

Comment:
'Default' is misspelled.

Reply:
Updated [here](https://github.com/cerner/terra-core/pull/1306/commits/6506e16da9f3b3d557d3c71bb7fa03a4874a99a9).